mtgjson4/outputter.py

```diff
@@ -4,12 +4,13 @@
 import contextvars
 import json
 import logging
-from typing import Any, Dict, List
+from typing import Any, Dict, List, Set
 
 import mtgjson4
-from mtgjson4 import util
+from mtgjson4 import util, SUPPORTED_FORMAT_OUTPUTS
```

Comment:
Is this necessary? `mtgjson4.SUPPORTED_FORMAT_OUTPUTS` should work inline.

Reply:
A matter of style maybe? Do we generally want these constants to be prefixed with the module name?

Reply:
We don't have a style doc for such a thing, but I'm fine with mtgjson4.variable (see my earlier comment about GLOBAL class coming soon :tm: )